tests/e2e-tests/config/jest.setup.js

```diff
@@ -1,7 +1,12 @@
 /** format */
```

Comment:
Perhaps worth looking into using the config library to setting timeouts? Again not a blocker.

Reply:
Will look into it in the next PR as well. Thanks for looking for ways to optimize.